### PR TITLE
Add OmnibusF4 V6 target

### PR DESCRIPTION
**Pull request by asizon**
I have created target OmnibusF4 V6 as a variant of FireworksV2

### CODE REVIEW

**Comment by asizon** [comment]
[inav_2.0.1_OMNIBUSF4V6.zip](https://github.com/iNavFlight/inav/files/2855350/inav_2.0.1_OMNIBUSF4V6.zip)
.HEX for test  @dl1bg

**Comment by asizon** [comment]
@shellixyz What has happened in this PR?take advantage to put past commits?

**Comment by asizon** [comment]
lol, I understand, i have open PR again master and now is mixed.
It does not matter if the master merged commits merges with the development, right? or should I backtrack?

**Comment by shellixyz** [comment]
You have to use the development branch as a base. I just changed the destination branch from master to development. Rebase your branch onto development and force push.

**Comment by dl1bg** [comment]
Thank you, Asizon!
Just updated. The I2C magnetometer (HMC5883) if the BM-880 is not detected.
It works with Andrew's (atobiese's) version
https://www.dronetrest.com/t/i2c-magnetometer-not-working-on-omnibus-f4-v6/7005/11?u=fat

# INAV/OMNIBUSF4V6 2.0.1 Feb 12 2019 / 11:41:25 (da78e89ff)

Did I miss to configure something?

File: src/main/target/FIREWORKSV2/target.h [hunk 1]
```diff
@@ -173,8 +196,13 @@
 #define SERIALRX_UART                   SERIAL_PORT_USART1
 #define SMARTAUDIO_UART                 SERIAL_PORT_USART4
 
+#define TARGET_CONFIG
```
Comment (digitalentity) [inline src/main/target/FIREWORKSV2/target.h:199]: This is not necessary if you don't have target-specific config code.

**Review by digitalentity — COMMENTED** [review]
Also, please add to `release.mk` list

**Comment by dl1bg** [comment]
Now tried the two versions back and forth, I2C does not work with this HEX file.
Please tell me if I can help in any way.

**Comment by asizon** [comment]
[inav_2.1.0_OMNIBUSF4V6.zip](https://github.com/iNavFlight/inav/files/2862126/inav_2.1.0_OMNIBUSF4V6.zip)
@dl1bg try it please :)

**Comment by asizon** [comment]
the new PR atempt #4369

**Comment by CRISTSC** [comment]
the pin assignment is not entirely right for this board. this is what i discovered with a multi-meter: 
# resource
IO:
----------------------
**A00: RSSI IN**
**A01; SERIAL4 UART RX 
A02: SERIAL2 UART TX** 
A03: MOTOR3 OUT
A04: MPU CS
A05: SPI1 SCK
A06: SPI1 MISO
A07: SPI1 MOSI
**A08: FREE**
A09: SOFTSERIAL1 UART TX/RX
A10: FREE 
A11: USB IN
A12: USB OUT
A13: FREE 
A14: FREE 
A15: OSD CS
B00: MOTOR1 OUT
B01: MOTOR2 OUT
B02: FREE 
B03: BARO CS
B04: BEEPER OUT
B05: MOTOR4 OUT
**B06:  LED1 OUT** 
**B07: CAM_C** 
B08: I2C1 SCL
B09: I2C1 SDA
B10: SERIAL3 UART TX
B11: SERIAL3 UART RX
B12: FLASH CS
B13: SPI2 SCK
B14: SPI2 MISO
B15: SPI2 MOSI
C00: FREE 
C01: ADC CH1
C02: ADC CH2
C03: FREE 
C04: FREE 
C05: FREE 
C06: FREE 
C07: SERIAL6 UART RX
**C08: MOTOR 5 OUT 
C09: MOTOR 6 OUT**
C10: SPI3 SCK
C11: SPI3 MISO
C12: SPI3 MOSI
C13: FREE 
C14: FREE 
C15: FREE 
D00: FREE 
D01: FREE 
D02: SPI_PREINIT CS
D03: FREE 
D04: FREE 
D05: FREE 
D06: FREE 
D07: FREE 
D08: FREE 
D09: FREE 
D10: FREE 
D11: FREE 
D12: FREE 
D13: FREE 
D14: FREE 
D15: FREE

i bold what i discovered wrong so far. can you please make the modifications to recover the full functionality of this board. thanks

**Pull request closed without merging**